api-reference/beta/toc.yml

```
@@ -22032,6 +22032,39 @@ items:
               href: api/agreement-post-files.md
             - name: List localized files
               href: api/agreement-list-files.md
+    - name: Recommendations
```

What are your thoughts:
- Move this to "Identity and access" > "Directory management"
- Make "Recommendations" and "Impacted resources" peers in the TOC, instead of nesting.
```suggestion
    - name: Directory recommendations
```